.github/workflows/integrate-dm.yaml

```diff
@@ -66,6 +66,23 @@ jobs:
           # ref: https://stackoverflow.com/questions/43099116/error-the-input-device-is-not-a-tty
           docker exec tiup-cluster-control bash /tiup-cluster/tests/tiup-dm/run.sh ${{matrix.cases}}
 
+      - name: Collect component log
```

Comment:
Why upload log? I think just replacing `cat ./tests/*.log` with a correct log path will be fine.

Reply:
As cluster https://github.com/pingcap/tiup/blob/cc41ad5af87bde821ecde48d63dd7c1c40d72ea2/.github/workflows/integrate-cluster-cmd.yaml#L70 uploads it, so I think dm should upload it too.

Reply:
OK, How about add `cat /path/to/*.log` too? By this way we can check what happened without downloading the logs when some error occurs in test.

Reply:
I don't think it's a good idea, as the debug file may be very large and may not be human-readable on the terminal, BTW, I'll give it a try.

Reply:
I'm going to adjust my strategy to look like this:if the test passed, no log were uploaded and print out; else upload and print it.

Reply:
That's cool